matRad_calcParticleDose.m

```diff
@@ -85,22 +85,35 @@
 if strcmp(pln.bioOptimization,'effect') || strcmp(pln.bioOptimization,'RBExD') ... 
         && strcmp(pln.radiationMode,'carbon')
     fprintf('matRad: loading biological base data... ');
-    mTissueClass = zeros(size(V,1),1);
+    vTissueIndex = zeros(size(V,1),1);
+    
+    %set overlap priorities
+    cst  = matRad_setOverlapPriorities(cst);
+    
     for i = 1:size(cst,1)
         % find indices of structures related to V
         [~, row] = ismember(cst{i,4},V,'rows');  
-        if ~isempty(cst{i,5}) && isfield(cst{i,5},'TissueClass')
-            mTissueClass(row) = cst{i,5}.TissueClass;
+        if ~isempty(cst{i,5}) && isfield(cst{i,5},'alphaX') && isfield(cst{i,5},'betaX')
+            
+            IdxTissue = find(ismember(machine.data(1).alphaX,cst{i,5}.alphaX) & ...
+                                     ismember(machine.data(1).betaX,cst{i,5}.betaX));
+                                 
```

Comment:
I think this check is pretty bullet proof! Do we really need the consistency check a couple of lines further below?